.buildkite/pipeline.elastic-agent-package.yml

```diff
@@ -99,6 +99,16 @@ steps:
           .buildkite/scripts/steps/package.sh
         artifact_paths:
           - "build/distributions/**/*"
+  
+  - label: ":elastic-stack: Trigger Elastic Agent msi build"
+    trigger: "elastic-stack-installers"
+    build:
+      branch: main
```

Comment:
Do we intend to backport this? If we do the branch name needs to change correct? How will that be kept up to date as new release branches are created?

Reply:
This was just a draft PR to see if the trigger would work, the branch value will indeed be changed to rely on the current branch the pipeline is running on. I'll close this PR for now until the .msi PR is merged.